abs.yaml

```diff
@@ -0,0 +1,4 @@
+upload_channels:
```

Comment:
Do we still need this or can it be deleted?

Reply:
I removed it